index.html

```diff
@@ -0,0 +1,208 @@
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="assets/css/style.css" />
+    <title>Home Page - ReviewMe</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navbar fixed-top navbar-expand-md navbar-light bg-light">
+        <div class="container-fluid">
+          <a class="navbar-brand nav-bar-logo-image" href="index.html">
+            <img
+              src="assets/images/reviewme-website-favicon-color-updated.png"
+              alt="logo image"
+            />
+          </a>
+          <button
+            class="navbar-toggler collapsed d-flex d-md-none flex-column justify-content-around"
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbarNav"
+            aria-controls="navbarNav"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <span class="toggler-icon top-bar"></span>
+            <span class="toggler-icon middle-bar"></span>
+            <span class="toggler-icon bottom-bar"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarNav">
+            <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+              <li class="nav-item me-5">
+                <a class="nav-link" aria-current="page" href="about.html"
+                  >About</a
+                >
+              </li>
+              <li class="nav-item me-5">
+                <a class="nav-link" href="#service-section">Services</a>
+              </li>
+              <li class="nav-item me-5">
+                <a class="nav-link" href="#partners-section">Sponsors</a>
+              </li>
+              <li class="nav-item me-5">
+                <a class="nav-link">Forum</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section class="banner-section">
+        <div class="background-image"></div>
+        <div class="banner-text-content banner-text-content-home-page">
+          <h1 class="mb-3">Hello Developers!</h1>
+          <h2 class="fw-bold h2-banner-home">
+            ReviewMe, GET INSTANT HELP, <br />
+            MEET PROFESSIONALS AND <br />
+            MOVE 5x FASTER
+          </h2>
+          <div class="site-description-div">
+            <p class="p-3">
+              Find qualified code reviewers, get feedback on your code, and
+              improve the quality of your code. Learn new skills, and give
+              feedback to the software development community.
+            </p>
+          </div>
+          <div class="site-date-div">
+            <p class="text-date">2023.07.05 (Wed.) ~ 07 (Fri.)</p>
+            <p class="text-date-description">Upcoming ReviewMe Hackathon</p>
+          </div>
+        </div>
+      </section>
```

Comment:
- [ ] Kindly make sure to follow the design guidelines as much as possible, for this section, kindly address the following:-

    - You need to use the proper font, and font size to match the actual design.
    -  You need to add the proper paddings, spacings, and margins to adjust the layout.

| Your design | Actual design |
| ------------- | ------------- |
| ![image](https://github.com/kambereBr/ReviewMe/assets/102612396/181da086-2c6a-497c-a27b-574d94c3db72)  | ![image](https://github.com/kambereBr/ReviewMe/assets/102612396/d5edf828-a8c3-43b6-b671-491decbb43f5) |